schematics/types/base.py

```diff
@@ -965,8 +966,39 @@ def to_primitive(self, value, context=None):
         else:
             value = value.astimezone(self.UTC)
         delta = value - self.EPOCH
-        ts = (delta.days * 24 * 3600) + delta.seconds + delta.microseconds / 1E6
-        return ts
+        return delta.total_seconds()
```

Comment:
This is not related to the fix itself, but is a way more explicit way to get this value.